lms/static/js/instructor_dashboard/ecommerce.js

```diff
@@ -31,5 +31,27 @@ var edx = edx || {};
             minDate: 0
         });
         var view = new edx.instructor_dashboard.ecommerce.ExpiryCouponView();
+        var request_response = $('.reports .request-response');
+        var request_response_error = $('.reports .request-response-error');
+        $('input[name="user-enrollment-report"]').click(function(){
+            var url = $(this).data('endpoint');
+            //return location.href = url;
+            $.ajax({
+             dataType: "json",
+             url: url,
+             success: function (data) {
+                request_response.text(data['status']);
+                return $(".reports .msg-confirm").css({
+                  "display": "block"
+                });
+               },
+             error: function(std_ajax_err) {
+                request_response_error.text("${_('Error generating grades. Please try again.')}");
```

Comment:
Is `_` here the underscore.js or gettext? You can avoid confusion by using `gettext`.